client/landing/stepper/hooks/use-site.ts

```diff
@@ -17,6 +17,7 @@ export function useSite() {
 	const isRequestingSelectedSite = useSelector( ( state ) =>
 		isRequestingSite( state, siteIdOrSlug )
 	);
+	const lastRequestedSiteIdOrSlug = useRef( '' );
```

Comment:
We can also use the `usePrevious` hook to access the previous value 🙂

```js
import { usePrevious } from '@wordpress/compose';
```

Reply:
> We can also use the usePrevious hook to access the previous value 🙂

Nice suggestion, thanks!